server/src/diagnostics.ts

```diff
@@ -317,13 +317,14 @@ export class Diagnostics {
     sourceFile.importDeclarations.forEach((importDeclaration) => {
       if (!importDeclaration.node.loc) return
 
+      const range = this.rangeFromLoc(textDocument, importDeclaration.node.loc)
```

Comment:
Hey @nachiket87, thanks for taking a stab at this! 

Either we should add the diagnostic range only to the actual package name, or we look at the AST Node to figure out the import source node and use that loc for the replacement. Does that make sense?

Reply:
Yes, it does. I will try both approaches and see how it goes! thanks for your prompt reply 😄